### PR TITLE
Kitchen sink

### PR DESCRIPTION
**Pull request by shashankp**
https://github.com/fonsp/PlutoUtils.jl/pull/13#issuecomment-690677692

### CODE REVIEW

File: docker/precompiled/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+    && apt-get -y install clang \
+    && rm -rf /var/lib/apt/lists/*
 
 ENV USER pluto
 ENV USER_HOME_DIR /home/${USER}
+ENV DEBIAN_FRONTEND=noninteractive
 ENV JULIA_DEPOT_PATH ${USER_HOME_DIR}/.julia
 ENV NOTEBOOK_DIR ${USER_HOME_DIR}/notebooks
-ENV JULIA_NUM_THREADS = 100
+ENV JULIA_NUM_THREADS=100
```
Comment (lungben) [inline docker/precompiled/Dockerfile]: I have just seen that using "=" for variable assignment in Dockerfile does not work.
This line should be changed to
`ENV JULIA_NUM_THREADS 100`
Same for here:
`ENV DEBIAN_FRONTEND noninteractive`
For the non-precompiled Docker image I just corrected it.

**Review by lungben — APPROVED** [review]
Thanks!

File: docker/precompiled/Dockerfile (hunk 1 above)
Comment (shashankp) [reply docker/precompiled/Dockerfile]: thanks! no wonder it wasn't picking up 😅

**Comment by shashankp** [comment]
Docker repo name is to be updated from `shashankpv/` to `plutojl` in the Dockerfiles & Github Actions' `.github/workflows/docker-publish.yml`

**Pull request merged**